exercise-2/exercise-2-10/exercise-2-10.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Volume of a Sphere</title>
+    <style> 
+      body{padding-top:30px;}
+      label,input{display:block;} 
+    </style>
+  </head>
+  <body>
+    <p>Input radius value and get the volume of a sphere.</p>
+    <form action method="post" id="MyForm">
+      <label for="radius">Radius</label>
+      <input type="text" name="radius" id="radius" required>
```

Comment:
I see you use `parseInt` the input value. We should use number input or at least have a number pattern for text input